spyder/app/mainwindow.py

```diff
@@ -884,6 +884,11 @@ def create_edit_action(text, tr_text, icon):
         from spyder.plugins.editor.lsp.manager import LSPManager
         self.lspmanager = LSPManager(self)
 
+        # Fallback completion thread
+        self.set_splash(_("Creating fallback completion engine"))
+        from spyder.plugins.editor.utils.fallback import FallbackActor
+        self.fallback = FallbackActor(self)

```

Comment:
Please call this attribute `fallback_completions` instead of `fallback` so it's easier to understand.